release/build.sh

```diff
@@ -62,17 +62,17 @@ ${DEPENDENCIES:-$(cat <<EOD
     auto: modules
   client-go:
     git: https://github.com/istio/client-go
-    branch: master
+    branch: release-1.9
     goversionenabled: true
   gogo-genproto:
     git: https://github.com/istio/gogo-genproto
-    branch: master
+    branch: release-1.9
   test-infra:
     git: https://github.com/istio/test-infra
-    branch: master
+    branch: release-1.9
```

Comment:
this should be master  still, we don't branch test-infra

Reply:
Changed